tests/foreman/ui/test_operatingsys.py

```diff
@@ -394,12 +394,12 @@ def test_update_os_template(self):
         """
         os_name = gen_string("alpha", 4)
         template_name = gen_string("alpha", 4)
-        os_attrs = entities.OperatingSystem(name=os_name).create_json()
+        os = entities.OperatingSystem(name=os_name).create()
         entities.ConfigTemplate(
             name=template_name,
-            operatingsystem=[os_attrs['id']],
+            operatingsystem=[os],
```

Comment:
Yes @oshtaier ... ConfigTemplate is expecting OperatingSystem object instead of ID. Otherwise it fails.